common/unified-chart/templates/web/web-deployment.yaml

```diff
@@ -83,6 +77,10 @@ spec:
           envFrom:
             - configMapRef:
                 name: {{ include "name" . }}
+            {{- if .Values.additionalSecrets }}
+            - secretRef:
+                name: additional-{{ include "secret.name" . }}
+            {{- end }}
```

Comment:
it would be better to do this:
range .Values.web.additionalEnvFrom
  tpl . $top

Reply:
but need to add this to hp/worker as well